packages/browser-policy/README.md

```diff
@@ -31,10 +31,12 @@ For most apps, we recommend that you take the following steps:
 * Add `browser-policy` to your app to enable a starter policy. With this starter
 policy, your app's client code will be able to load content (images, scripts,
 fonts, etc.) only from its own origin, except that XMLHttpRequests and WebSocket
-connections can go to any origin. Further, your app's client code will not be
-able to use functions such as `eval()` that convert strings to code. Users'
+connections can go to any origin. Users'
 browsers will only let your app be framed by web pages on the same origin as
-your app.
+your app. Further, your app's client code will not be
+able to use functions such as `eval()` that convert strings to code. 
+However, note that if you also use the `dynamic-imports` package, this limitation 
```

Comment:
I think this maybe be a bit misleading. It could be read like that if you use `dynamic-imports` package it somehow enables a workaround which allows one to use `eval` despite it being disabled by the policy, not that it disables the "prevent eval` policy.